mne_bids_pipeline/_run.py

```diff
@@ -277,14 +279,27 @@ def clear(self) -> None:
         self.memory.clear()
 
 
-def save_logs(*, config: SimpleNamespace, logs) -> None:  # TODO add type
+def save_logs(*, config: SimpleNamespace, logs: list[pd.Series]) -> None:
     fname = config.deriv_root / f"task-{get_task(config)}_log.xlsx"
 
     # Get the script from which the function is called for logging
     sheet_name = _short_step_path(_get_step_path()).replace("/", "-")
     sheet_name = sheet_name[-30:]  # shorten due to limit of excel format
 
-    df = pd.DataFrame(logs)
+    # We need to make the logs more compact to be able to write Excel format
+    # (32767 char limit)
```

Comment:
I don't understand this, how does compressing the logs help with writing the (then-uncompressed) data to Excel?

Reply:
Xlsx has a 32767 chat limit for each cell. The cell data for entries in what was formerly the `cfg` column in the spreadsheet gets compressed both in json characters (remove indentation and unnecessary whitespace) and then with zlib. This compressed data is now written in a cfg_zlib column instead

Reply:
does that mean that it's not human-readable anymore?